geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -726,13 +733,13 @@
   public static final String CREATE_INDEX__REGION = "region";
   public static final String CREATE_INDEX__REGION__HELP =
       "Name/Path of the region which corresponds to the \"from\" clause in a query.";
-  public static final String CREATE_INDEX__MEMBER = "member";
+  public static final String CREATE_INDEX__MEMBER = MEMBER;
```

Comment:
I am wondering if you can delete all these other xxx_GROUP or xxx_MEMBER string declarations and change all reference of them to use the generic ones.